src/Http/Requests/Speaker/Rules/GithubLinkRules.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Http\Requests\Speaker\Rules;
+
+use Blumilk\Meetup\Core\Http\Requests\BaseRules;
+
+class GithubLinkRules extends BaseRules
+{
+    protected static array $rules = [
+        "regex:/http(s)?:\/\/(www\.)?github\.com\/.+/i",
```

Comment:
Maybe `starts_with` will be better than this regex
https://laravel.com/docs/9.x/validation#rule-starts-with